src/GraphQL/Validation/Rules/UniqueInputFieldNames.cs

```diff
@@ -20,9 +20,9 @@ public class UniqueInputFieldNames : IValidationRule
 
         /// <inheritdoc/>
         /// <exception cref="UniqueInputFieldNamesError"/>
-        public Task<INodeVisitor> ValidateAsync(ValidationContext context)
-        {
-            return new NodeVisitors(
+        public Task<INodeVisitor> ValidateAsync(ValidationContext context) => _nodeVisitor;
```

Comment:
@Shane32 we forgot to make a static field in this class 🤦

Reply:
I'm glad you're taking another pass through it